src/server/session.ts

```diff
@@ -1904,6 +1905,71 @@ export class Session<TMessage = string> implements EventSender {
         });
     }
 
+    private mapCode(args: protocol.MapCodeRequestArgs): protocol.FileCodeEdits[] {
+        const formatOptions = this.getHostFormatOptions();
+        const preferences = this.getHostPreferences();
+        const projects = new Map<Project, MapCodeDocumentMapping[]>();
+        args.mappings.forEach(mapping => {
+            if (!mapping.file) {
+                // TODO(@zkat): We don't support mapping file-less code yet.
+                return;

```

Comment:
Should this be a `continue` instead so that other mappings are still processed?

Reply:
That's a `forEach`. The others will get processed, no? Did I miss something?

Reply:
Oh duh nvm.